bundlewrap/items/directories.py

```diff
@@ -252,8 +254,8 @@ def sdict(self):
             return {
                 'type': 'directory' if path_info.is_directory else path_info.stat['type'],
                 'mode': path_info.mode,
-                'owner': path_info.owner,
-                'group': path_info.group,
+                'owner': ('+' + path_info.owner_id) if use_uid else path_info.owner,
```

Comment:
A thought that ran through my head, but is in no means a "please do that" thing:

We could modify `sdict` so it'll always return both owner/group name and uid/gid, then in `cdict` return either name or id, depending on what the user specified.
This would allow users to be able to easily spot whether a id or name will be changed, instead of having to know "ah, it's prefixed with a plus sign, so it's an id".

What do you think, @vain @MaZderMind?

Reply:
I’d be fine with both, to be honest. Using the plus sign is probably pretty rare, so I’d say it’s within reason to consult the docs in those cases.

(And the plus sign has the slight advantage of being unambiguous. There’s no potential conflict when you specify both `uid` and `owner`.)

But as I said, I’d be fine with both approaches.